drivers/supplemental_enrollment_data/README.md

```diff
@@ -0,0 +1,3 @@
+## SupplementalEnrollmentData README
+
+This README file should be used to explain the functionality of the driver.
```

Comment:
Is this driver intended to incorporate ALL the supllemental enrollment data models, or just the TPC one? I'd prefer that it be just the TPC one, in which case, should it be renamed, and described here?